app/models/requests/request.rb

```diff
@@ -313,9 +313,7 @@ def build_requestable_mfhd_item(_requestable_items, holding_id, item, barcodesor
           holding: calculate_holding,
           location: current_location
         )
-        # sometimes availability returns items without any status
-        # see https://github.com/pulibrary/marc_liberation/issues/174
-        Requests::Requestable.new(params) # unless item["status"].nil?
+        Requests::Requestable.new(params) unless item['on_reserve'] == 'Y'
```

Comment:
@maxkadel what do you think of moving the condition in line 303? 
`return if item['on_reserve'] == 'Y'` or `return unless item['on_reserve'] == 'N'`
before we start calculating the rest.

Reply:
Good call - done!